src/components/Avatar/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+function Avatar({url}) {
+    return (
+        <img src={url} className="user-avatar" alt="github-user-avatar"/>
+    )
+}
+
+export default Avatar
```

Comment:
Great addition to allow users to see their avatar on loading.  Makes it feel more personalised!